programmers/4일차/완주하지-못한-선수.js

```diff
@@ -0,0 +1,39 @@
+function solution(participant, completion) {
+    const objectCompletion = {};
+    
+    for (let  person of participant) {
+        objectCompletion[person] = objectCompletion[person] ? objectCompletion[person] + 1 : 1;
```

Comment:
```suggestion
        if (person in objectCompletion) objectCompletion[person] += 1;
        else objectCompletion[person] = 1
```
너무 잘 푸셨어요  이건  피드백은 아니고  in 연산자를 이용하면  객체에  해당 값이 포함되어있는지 판단할 수 있다는 걸  알려드리고 싶어서  써봤습니다  지금 푸신 방법도 좋은 방법입니다